mlflow/utils/autologging_utils.py

```diff
@@ -1,1360 +1,33 @@
-import os
```

Comment:
For modularity, the previous contents of `autologging_utils` have been moved to `mlflow/utils/autologging/__init__`.   `autologging_utils` now imports select contents from the new `mlflow/utils/autologging/__init__` file to preserve compatibility with other modules that import from `mlfow.utils.autologging_utils`.

I will file a follow-up PR that drops this `autologging_utils` file and changes existing imports to import from `mlflow.utils.autologging` instead of `mlflow.utils.autologging_utils`.